extensions/vscode/src/granite/panels/setupGranitePage.ts

```diff
@@ -304,6 +304,9 @@ export class SetupGranitePage {
           case "installOllama":
             await this.installOllama(data.mode, panel);
             break;
+          case "openUrl":
+            await this.openUrl(data);
```

Comment:
When you click on the Open tutorial button, it no longer marks the step as done. So you have 2 ways of fixing it.

1. Instead of adding this new `openUrl` command, just keep the existing `showTutorial` call, and replace 
> await this.showTutorial();

with

> await this.openUrl("https://docs.granitecode.ai/getting-started.html");

2. On the GUI side, post a message that sets this.wizardState.stepStatuses[FINAL_STEP] = true; then open the URL.


The 1st solution seems the simplest by far, so I'd go with that.